pkg/manager/member/tikv_member_manager.go

```diff
@@ -515,6 +515,17 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		VolumeMounts: volMounts,
 		Resources:    controller.ContainerResource(tc.Spec.TiKV.ResourceRequirements),
 	}
```

Comment:
@hhy5861 Could you please also port the change for pkg/manager/member/tidb_discovery_manager.go in #3821?
It's OK to roll update discovery, no need to add control parameter for it.